### PR TITLE
Release artifacts rework

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

N/A

# Overview

- d953498e2b4c1e16d493f7aaf948ad0bdb57963b
  :round_pushpin: **Alternate path: download byron binaries in CI**
  
- d71374ca87082b6e5b4f11314f99efa726d2efa6
  :round_pushpin: **adjust names of exported artifacts in Travis**

# Comments

### CODE REVIEW

**Review by Anviking — APPROVED** [review]
Bash interpreted `$TRAVIS_TAG_macos64` as one variable. Changed to `$TRAVIS_TAG-macos64` in  a439ab7. 

Works on my machine.

**Comment by KtorZ** [comment]
> Bash interpreted $TRAVIS_TAG_macos64 as one variable. 

:man_facepalming:  ... Okay then.

File: test.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
+############################
+# cardano-wallet-byron
+############################
+
+# Fetch the linux archive
+curl -L https://hydra.iohk.io/job/Cardano/cardano-wallet/cardano-wallet-byron-linux64/latest/download/1 | tar xz
```
Comment (Anviking) [inline test.sh:7]: Hm… I shouldn't have committed this  😅

Comment (KtorZ) [reply test.sh:7]: Whoops. I didn't even see it :no_mouth:

**Pull request merged**